src/helpers.coffee

```diff
@@ -4,13 +4,15 @@ module.exports =
   octoHelpers:
     createLink: (path) ->
       url.resolve(process.env.HUBOT_OCTO_ENDPOINT, path)
+    slugIt: (name) ->
+      name.toLowerCase().replace(/[\. \s]/g, '-')
   httpHelpers:
     getBody: (response) ->
       new Promise (resolve, reject) ->
         resolve response.body
     validateSuccess: (response) ->
       new Promise (resolve, reject) ->
-        if response.status == 200
+        if response.status >= 200 || response.status < 300
```

Comment:
this will capture all successful status code (ie 2xx).
AFAIK all success response should be handled the same.

I see no point in having separate logic for each or the need to identify the difference.